.github/workflows/prepare-release.yaml

```diff
@@ -0,0 +1,61 @@
+name: Prepare Release
+
+on:
+  push:
+    branches:
+      - release/*
+
+jobs:
+  prepare-release:
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PYTHONPATH: /opt/hostedtoolcache/Python/3.9.14/x64/lib/python3.9/site-packages
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: refs/heads/main
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.9.4
+
+      - name: Setup Python environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9.14'
+
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip gitpython ruamel_yaml
+
+      - name: Generate changelog and Chart annotations
+        run: |
+          echo "[INFO]: Preparing release ${GITHUB_REF_NAME##*/}"
+          python src/main/scripts/prepare_release.py ${GITHUB_REF_NAME##*/}
+
+      - name: Configure Git
+        run: |
+          git config --global user.name "$GITHUB_ACTOR"
+          git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Lint yamls with ah cli
```

Comment:
We need to make sure ArtifactHub.io will like the annotations and the new version will be successfully parsed.

Reply:
Nice!